pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -269,10 +269,10 @@
     ConsumerBuilder<T> subscriptionMode(SubscriptionMode subscriptionMode);
 
     /**
-     * Sets a {@link MessageListener} for the consumer
+     * Sets a {@link MessageListener} for the consumer.
```

Comment:
There's a mix of third-person verbs (eg,. `Sets`) and infinitives (eg. `Select`).
I think it should be made consistent.